### PR TITLE
add to_dict method to Origins class

### PR DESCRIPTION
**Pull request by Solaxun**
Signed-off-by: solaxun <markcwoodworth@gmail.com>

This PR addresses #85 , and adds a to_dict method to the Origins class.

### CODE REVIEW

File: classes/origins.py [hunk 1]
```diff
@@ -53,3 +53,6 @@ def is_empty(self):
                     empty = False
                     break
         return empty
+
+    def to_dict(self):
+        return [origin.to_dict() for origin in self.origins]
```
Comment (nishakm) [inline classes/origins.py]: Could you add a line about this method to the documentation above?

**Comment by nishakm** [comment]
@Solaxun thank you! Could you add a body to your commit message (it can be from the issue) and also "Resolves #<issue number>"?

**Comment by Solaxun** [comment]
I'll have to sign-off later this evening - right now I'm in the office so any changes I'm making are in the browser, and I haven't figured out if it's possible to edit a commit body to add signoff from github (not using CLI).

**Comment by nishakm** [comment]
@Solaxun yup you can edit the commit body to add your Signed-off-by. On the command line run:
`git commit --amend -s` and it will automatically add the Signed-off-by. Then just do `git push -f origin master` to your fork and the PR will automatically get updated. Thanks for working on this!

**Comment by Solaxun** [comment]
Right -  it's just that I don't have access to the CLI while in the office, and it doesn't look like it's possible through the browser.  I'll make the change tonight when I get home, and no problem happy to work on it - thanks for providing a good entry point for new contributors!  I'll watch the repo for similar issues going forward.

**Pull request closed without merging**